packages/backend-common/src/compat/legacy/legacy.ts

```diff
@@ -64,7 +72,64 @@ function wrapTokenManager(tokenManager: TokenManager, auth: AuthService) {
       }
       await tokenManager.authenticate(token);
     },
-  } satisfies TokenManager;
+  };
+}
+
+/**
+ * Originally IdentityApi from `@backstage/plugin-auth-node`, not re-declared here for backwards compatibility
```

Comment:
```suggestion
 * Originally IdentityApi from `@backstage/plugin-auth-node`, re-declared here for backwards compatibility
```

or am i reading this wrong? 😅

Reply:
xD